boards/arm/ubx_bmd380eval_nrf52840/ubx_bmd380eval_nrf52840.yaml

```diff
@@ -0,0 +1,24 @@
+identifier: ubx_bmd380eval_nrf52840
+name: UBX_BMD380EVAL_NRF52840
+type: mcu
+arch: arm
+ram: 256
+flash: 1024
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
```

Comment:
Are you sure about this one?

Reply:
I guess it doesn't hurt, since it's just Crosstool-NG, which anyone can build from source.

Reply:
This board is nearly identical to the nrf52840dk_nrf52840, minus a couple GPIO pins. The same toolchain is in that yaml file.